week2/1-tuesday/class/exercise9/exercise9.js

```diff
@@ -0,0 +1,9 @@
+function tenAndTwenty(age){
+    if(age > 10 && age < 20){
```

Comment:
Well, done. See if you can add the error check on your own for this one (the code that makes sure it's actually dealing with a number. One thing to keep in mind on this is if someone enters a "5", for example, it's technically a string but you can still use it as long as you can cast it to a number (which you can in this case).